app/assets/javascripts/member-facing/backbone/sweet_placeholder.js

```diff
@@ -0,0 +1,62 @@
+const SweetPlaceholder = Backbone.View.extend({
+
+  el: '.sweet-placeholder',
+
+  events: {
+    'focus .sweet-placeholder__field': 'focus',
+    'blur  .sweet-placeholder__field': 'blur',
+    'click .sweet-placeholder__label': 'fauxcus',
+    'change .sweet-placeholder__field': 'decide',
+    'input .sweet-placeholder__field': 'decide',
+  },
+
+  initialize() {
+    for (var el of this.$el.find('.sweet-placeholder__field')) {
```

Comment:
Yeah I checked and const does in fact work but I think it's confusing for the iterator so I'm going to use let